devops: Don't auto-install brew deps

max-sixty opened the pull request:
As discussed in https://github.com/PRQL/prql/pull/3333#issuecomment-1678374434

(need to have another look in the morning)

max-sixty committed:
devops: Don't auto-install brew deps

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

eitsupi commented on Taskfile.yml:
```diff
@@ -26,6 +26,12 @@ vars:
+    hugo
+    npm
+    python3
+    elixir
```
I'm wondering if it's worth including elixir here.
I don't think prql-elixir has been substantially edited in the last few months.

max-sixty replied:
Yes OK

max-sixty committed:
Try falling back to `cargo install` for binstall

max-sixty committed:
Try adding the path?

Couldn't seem to find it; ref https://github.com/PRQL/prql/actions/runs/5870401451/job/15917365395?pr=3346#step:6:985

max-sixty committed:
Revert "Try adding the path?"

This reverts commit e52643cc89962a062a7139fa6fac5b03c8e46b11.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

max-sixty committed:
Merge branch 'main' into taskfile

max-sixty commented on Taskfile.yml:
```diff
@@ -22,17 +22,34 @@ vars:
+    wasm-bindgen-cli
+    wasm-opt
     wasm-pack
+  # Excluding `elixir` atm given it's not enabled on Mac and not yes supported
```
```suggestion
  # Excluding `elixir` atm given it's not enabled on Mac and currently unsupported
```

max-sixty committed:
Update Taskfile.yml

max-sixty commented on Taskfile.yml:
```diff
@@ -22,17 +22,34 @@ vars:
       - sh: which clang
         msg: |
+
+
```
```suggestion
```

max-sixty committed:
Update Taskfile.yml
